### DISCUSSION TITLE
Welcome to pycausalgps Discussions!

### DISCUSSION THREAD

**Discussion by Naeemkh** [NSAPH-Software/pycausalgps#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by liujunyug** [comment]
Thank you for your library designed for GPS. I am a learner in statistics. When I ran the example code, I encountered an error message stating that it could not import the name 'GeneralizedPropensityScore'. How can I acquire this function? Thanks！
![20250505130155](https://github.com/user-attachments/assets/a2bcc52b-4bde-4584-af4d-4e5e153a28ff)
